bindings/nodejs/lib/types/block/output/feature.ts

```diff
@@ -29,6 +30,9 @@ abstract class Feature {
  * Sender feature.
  */
 class SenderFeature extends Feature {
+    @Type(() => Address, {
+        discriminator: AddressDiscriminator,
+    })
```

Comment:
I have a hard time wrapping my head around this. Why is this not attached to `getSender`? This seems a bit weird just here in the SenderFeature

Reply:
`getSender` isn't really an issue, you can imagine that `address` is public, so when` SenderFeature` is constructed, `address` was assigned to an instance of `Object` with this annotation, now it's assigned to one of the `Address` classes. `getSender` just returns whatever is stored in `address`.

Reply:
This wouldn't have worked if there was more fields than just address, right?

Reply:
I think in that case, each field must have its own annotation.

Reply:
Thanks, that helped